pwiz_tools/Skyline/TestFunctional/DdaSearchTest.cs

```diff
@@ -109,6 +109,13 @@ public void TestDdaSearch()
         [TestMethod]
         public void TestDdaSearchMsgfPlus()
         {
+            if (Helpers.RunningResharperAnalysis)
```

Comment:
Let's have a function, so that it looks like this:

if (SkipForResharperAnalysis())
    return;